api/src/main/java/javax/enterprise/inject/spi/ProcessBeanAttributes.java

```diff
@@ -92,4 +92,18 @@
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public void veto();
+
+
+    /**
+     * Instructs the container to ignore all non-static, final methods with public, protected or default visibility declared on
+     * any bean type of the specific bean during validation of injection points that require proxyable bean type.
+     * These methods shouldn't be invoked on the bean instances.
+     *
+     * Calling this method will bypass standard rules for unproxyable bean types (section 3.11 of the spec)
```

Comment:
I would leave out this sentence. Also if referencing a spec section it might be better to use `<a>` hyperlink so that it's easier to find/refactor links if needed.